apps/examples/stack_protection/stack_protection_main.c

```diff
@@ -46,29 +83,66 @@ int stack_prot_main(int argc, char *argv[])
 	printf("Stack Overflow Protection testcase!!\n");
 
 	while (1) {
-		printf("\nPress N - Test Stack access within range\n");
-		printf("Press O - Test Stack access in overflow\n");
+		printf("\nPress M - Test Stack protection with MPU\n");
+#if defined(CONFIG_ARCH_ARMV8M_FAMILY)
```

Comment:
I think CONFIG_REG_STACK_OVERFLOW_PROTECTION is right.

Reply:
changed